mobile/lib/features/welcome/welcome_screen.dart

```diff
@@ -36,75 +39,182 @@ class _WelcomeScreenState extends State<WelcomeScreen> {
-                            context.go(LoadingScreen.route);
-                          } catch (e) {
-                            showSnackBar(ScaffoldMessenger.of(context), "$e");
-                          }
-                        }
-                      },
-                      child: const Text(
-                        'Continue',
-                        style: TextStyle(fontSize: 16),
+                  ),
+                  Container(
+                    decoration: BoxDecoration(
+                        color: tenTenOnePurple.withOpacity(0.05),
+                        borderRadius: const BorderRadius.only(
+                            topLeft: Radius.circular(10.0), topRight: Radius.circular(10.0))),
+                    child: Padding(
+                      padding: const EdgeInsets.only(left: 20, right: 20, bottom: 40),
+                      child: Column(
+                        mainAxisAlignment: MainAxisAlignment.end,
+                        crossAxisAlignment: CrossAxisAlignment.end,
+                        children: [
+                          const SizedBox(height: 20),
+                          Container(
+                            padding: const EdgeInsets.all(15),
+                            decoration: BoxDecoration(
+                                color: tenTenOnePurple.shade300.withOpacity(0.2),
+                                borderRadius: BorderRadius.circular(10.0)),
+                            child: Row(
+                              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                              children: [
+                                const SizedBox(
+                                  width: 250,
+                                  child: Text("10101 is still in testing and may contain defects.",
+                                      softWrap: true),
+                                ),
+                                TenTenOneSwitch(
+                                    value: _betaDisclaimer,
+                                    onChanged: (value) => setState(() => _betaDisclaimer = value)),
+                              ],
+                            ),
+                          ),
+                          const SizedBox(height: 10),
+                          Container(
+                            padding: const EdgeInsets.all(15),
+                            decoration: BoxDecoration(
+                                color: tenTenOnePurple.shade300.withOpacity(0.2),
+                                borderRadius: BorderRadius.circular(10.0)),
+                            child: Row(
+                              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                              children: [
+                                const SizedBox(
+                                  width: 250,
+                                  child: Text(
+                                      "If I lose my seed phrase, my funds will be lost forever",
```

Comment:
🙃 Maybe not worth clarifying, but the user can just hold onto their phone without uninstalling the app and be safe.

```suggestion
                                      "If I lose my seed phrase and my device, my funds will be lost forever",
```